components/kyma-operator/cmd/operator/main.go

```diff
@@ -128,3 +143,28 @@ func parseBackoffIntervals(backoffIntervals string) ([]uint, error) {
 
 	return backoffIntervalsParsed, nil
 }
+
+func setupLogrus(logFile string) (*logrus.Logger, func(ch <-chan struct{}), error) {
+	// create the logger
+	logger := logrus.New()
+	// with Json Formatter
+	logger.Formatter = &logrus.JSONFormatter{}
```

Comment:
Hi there and thanks for the PR!
I think that a nice-to-have feature would be to set the log format with an env/flag, as the logs in a json format are quite unreadable

Reply:
done